src/vs/workbench/parts/files/electron-browser/fileActions.contribution.ts

```diff
@@ -396,7 +396,7 @@ MenuRegistry.appendMenuItem(MenuId.ExplorerContext, {
 		id: OPEN_FOLDER_SETTINGS_COMMAND,
 		title: OPEN_FOLDER_SETTINGS_LABEL
 	},
-	when: ContextKeyExpr.and(ExplorerRootContext, ExplorerFolderContext)
+	when: ExplorerRootContext
```

Comment:
@isidorn I was not sure about this condition, isn't a root always a folder too? same for the one below